src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -184,8 +184,22 @@ def validate_jwt_svid(self, token: str, audience: str) -> JwtSvid:
         Returns:
             JwtSvid: If the token and audience could be validated.
         """
```

Comment:
Missing Raises statement in the docstring.

Reply:
Good Catch! I'll merge this PR and add the docstring in the other open PR that has to be updated.
Thanks.